app/models/story.rb

```diff
@@ -571,6 +572,12 @@ def log_moderation
     all_changes = self.changes.merge(self.tagging_changes)
     all_changes.delete("unavailable_at")
 
+    # condition because of duplicating this logic in controller
+    if self.repost_description == "1"
```

Comment:
Can this be `if self.repost_description`?

Reply:
I think, only with some extra logic, because view helper method ```check_box``` which used for repost description feature has default value.
```ruby
 check_box(object_name, method, options = {}, checked_value = "1", unchecked_value = "0")
``` 
[Docs](https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-check_box)